package.json

```diff
@@ -24,9 +24,9 @@
         "typescript": "^5.0.2"
     },
     "scripts": {
-        "install": "node-gyp-build \"node scripts/download-CanBridge.mjs\"",
+        "install": "node scripts/download-CanBridge.mjs && node-gyp rebuild",
         "prepublishOnly": "node scripts/download-CanBridge.mjs && tsc && prebuildify --napi",
-        "pretest": "node-gyp-build && tsc",
+        "pretest": "node-gyp rebuild && tsc",
```

Comment:
Why the move away from `node-gyp-build`?

Reply:
I can write on this one.

I am pretty sure it was to not break any dependencies but now looking back at this one it does seem a bit silly. I think this was leftover on trying to fix some regression with https://github.com/prebuild/node-gyp-build/issues/71 which was very annoying to fix. I can bump versions to validate these changes.

Reply:
Oh, that's an unfortunate bug. There's a reason I never put spaces in my username.

Reply:
It wasn't exactly whitespace in the username but where VS is installed by default.

When encountering this bug it would regularly look for `C:\Program` not knowing it would be in `C:\Program Files\Microsoft Visual Studio\[year]\[edition]` which caused some headaches.

This regression looks to be fixed in `node-gyp-build` with PR https://github.com/prebuild/node-gyp-build/pull/72 but just waiting to be merged.

Reply:
Resolved in latest merge from our origin.